Deploy-CAM.ps1

```diff
@@ -3012,6 +3012,28 @@ function Write-Host-Warning() {
     Write-Host ("`n$message") -ForegroundColor Red
 }
 
+# Check resource group location for cores, if less than $neededCores exit the launch
+function Check-Location-Cores() {
+    param(
+        [parameter(Mandatory=$true)]
+        $rgLocation,
+        [parameter(Mandatory=$true)]
+        $neededCores
+    )
+    # "cores" is the Name.Value for "Total Regional vCPUs"
+    $totalRegionUsage = Get-AzureRmVMUsage -Location $rgLocation | Where-Object {$_.Name.Value -eq "cores"}
+    $availableCores = $totalRegionUsage.Limit - $totalRegionUsage.CurrentValue
+    if($availableCores -lt $neededCores){
+        if($availableCores -eq 1){
+            $areis = "is"
+        }else{
+            $areis = "are" 
+        }
+        Write-Host-Warning "Not enough vCPUs available in $rgLocation. $neededCores vCPUs are required, $availableCores $areis available."
```

Comment:
You could probably get rid of the areis if you rephrased this to something like "_ vCPUs are required (_ available)" or soemthing

Reply:
Fixed, now reeds "... _ vCPUs are required, _ available."